src/Microsoft.DotNet.Cli.Utils/CommandResolution/MSBuildProject.cs

```diff
@@ -124,7 +124,9 @@ public LockFile GetLockFile()
             var lockFilePath = GetLockFilePathFromProjectLockFileProperty() ??
                 GetLockFilePathFromIntermediateBaseOutputPath();
 
-            return new LockFileFormat().Read(lockFilePath);
+            return new LockFileFormat()
+                .ReadWithLock(lockFilePath)
```

Comment:
This method follows the naming convention of the NuGet methods that its mimicking: https://github.com/NuGet/NuGet.Client/blob/27645cbb0be2c84dd047c8287338920bd273688f/src/NuGet.Core/NuGet.Common/PathUtil/FileUtility.cs#L29